src/shogun/transfer/multitask/MultitaskKernelMaskNormalizer.h

```diff
@@ -222,7 +222,7 @@ class CMultitaskKernelMaskNormalizer: public CKernelNormalizer
 	/**
 	 * @return list of active task ids
 	 */
-	std::vector<int32_t> get_active_tasks()
+	std::vector<int32_t> get_active_tasks() const
```

Comment:
noexcept

Reply:
this function creates an `std::vector` and uses `push_back` which can throw exception